hack/make.sh

```diff
@@ -29,11 +29,22 @@ export MAKEDIR="$SCRIPTDIR/make"
 
 # We're a nice, sexy, little shell script, and people might try to run us;
 # but really, they shouldn't. We want to be in a container!
-if [ "$PWD" != "/go/src/$DOCKER_PKG" ] || [ -z "$DOCKER_CROSSPLATFORMS" ]; then
+INCONTAINER=1
+if [ "$(go env GOHOSTOS)" = 'windows' ]; then
+	if [ -z $FROM_DOCKERFILE ]; then
+		INCONTAINER=0
```

Comment:
If we drop the `0` here (`inContainer=`), we can just use `-z` for testing the value later. :smile: